<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calendarViewUI</class>
 <widget class="QWidget" name="calendarViewUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitterCalendarTab">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="handleWidth">
      <number>5</number>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QFrame" name="frmCalJobs">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>3</number>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QCalendarWidget" name="calMain">
         <property name="accessibleName">
          <string>calMain</string>
         </property>
         <property name="gridVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>300</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitterCalLists">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>5</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="childrenCollapsible">
          <bool>false</bool>
         </property>
         <widget class="QFrame" name="frmBuyJobList_2">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_15">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QToolBox" name="tboxCalJobs">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="currentIndex">
              <number>1</number>
             </property>
             <widget class="QWidget" name="tboxPageJobDay">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>282</width>
                <height>287</height>
               </rect>
              </property>
              <attribute name="label">
               <string>Jobs</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QLabel" name="lblCalPriceJobDay">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <property name="text">
                  <string>Scheduled income for the day:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="vmLineEdit" name="txtCalPriceJobDay">
                 <property name="font">
                  <font>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="vmListWidget" name="lstCalJobsDay"/>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tboxPageJobWeek">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>282</width>
                <height>287</height>
               </rect>
              </property>
              <attribute name="label">
               <string>Jobs</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_21">
               <item>
                <widget class="QLabel" name="lblCalPriceJobWeek">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <property name="text">
                  <string>Scheduled income for the week:</string>
                 </property>
                 <property name="scaledContents">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="vmLineEdit" name="txtCalPriceJobWeek">
                 <property name="font">
                  <font>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="vmListWidget" name="lstCalJobsWeek"/>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tboxPageJobMonth">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>282</width>
                <height>287</height>
               </rect>
              </property>
              <attribute name="label">
               <string>Jobs</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_22">
               <item>
                <widget class="QLabel" name="lblCalPriceJobMonth">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <property name="text">
                  <string>Scheduled income for the month:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="vmLineEdit" name="txtCalPriceJobMonth">
                 <property name="font">
                  <font>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="vmListWidget" name="lstCalJobsMonth"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QToolBox" name="tboxCalPays">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <widget class="QWidget" name="tboxPagePayDay">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>281</width>
             <height>287</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Payment</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="lblCalPricePayDay">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <property name="text">
               <string>Actual income for the day:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="vmLineEdit" name="txtCalPricePayDay">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <widget class="vmListWidget" name="lstCalPaysDay"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tboxPagePayWeek">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>281</width>
             <height>287</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Payment</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_17">
            <item>
             <widget class="QLabel" name="lblCalPricePayWeek">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <property name="text">
               <string>Actual income for the week:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="vmLineEdit" name="txtCalPricePayWeek">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <widget class="vmListWidget" name="lstCalPaysWeek"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tboxPagePayMonth">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>281</width>
             <height>287</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Payment</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_18">
            <item>
             <widget class="QLabel" name="lblCalPricePayMonth">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <property name="text">
               <string>Actual income for the month:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="vmLineEdit" name="txtCalPricePayMonth">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <widget class="vmListWidget" name="lstCalPaysMonth"/>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QToolBox" name="tboxCalBuys">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <widget class="QWidget" name="tboxPageBuyDay">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>282</width>
             <height>287</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Purchase</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QLabel" name="lblCalPriceBuyDay">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <property name="text">
               <string>Purchases for the day:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="vmLineEdit" name="txtCalPriceBuyDay">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <widget class="vmListWidget" name="lstCalBuysDay"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tboxPageBuyWeek">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>282</width>
             <height>287</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Purchase</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_23">
            <item>
             <widget class="QLabel" name="lblCalPriceBuyWeek">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <property name="text">
               <string>Purchases for the week:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="vmLineEdit" name="txtCalPriceBuyWeek">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <widget class="vmListWidget" name="lstCalBuysWeek"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tboxPageBuyMonth">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>282</width>
             <height>287</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Purchase</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_24">
            <item>
             <widget class="QLabel" name="lblCalPriceBuyMonth">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <property name="text">
               <string>Purchases for the month:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="vmLineEdit" name="txtCalPriceBuyMonth">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <widget class="vmListWidget" name="lstCalBuysMonth"/>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>vmLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">vmWidgets/vmwidgets.h</header>
  </customwidget>
  <customwidget>
   <class>vmListWidget</class>
   <extends>QListWidget</extends>
   <header location="global">vmWidgets/vmlistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
